lib/gui/models/drives.js

```diff
@@ -20,103 +20,94 @@
  * @module Etcher.Models.Drives
  */
 
-const angular = require('angular');
 const _ = require('lodash');
 const Store = require('./store');
-const MODULE_NAME = 'Etcher.Models.Drives';
-const Drives = angular.module(MODULE_NAME, []);
 
-Drives.service('DrivesModel', function() {
-
-  /**
-   * @summary Check if there are available drives
-   * @function
-   * @public
-   *
-   * @returns {Boolean} whether there are available drives
-   *
-   * @example
-   * if (DrivesModel.hasAvailableDrives()) {
-   *   console.log('There are available drives!');
-   * }
-   */
-  this.hasAvailableDrives = () => {
-    return !_.isEmpty(this.getDrives());
-  };
+/**
+ * @summary Check if there are available drives
+ * @function
+ * @public
+ *
+ * @returns {Boolean} whether there are available drives
+ *
+ * @example
+ * if (availableDrives.hasAvailableDrives()) {
+ *   console.log('There are available drives!');
+ * }
+ */
+exports.hasAvailableDrives = () => {
```

Comment:
Ahh, I see. Then the fix is to re-word the message. The function will throw if you pass nothing to it (which is what that test checks), but it will happily accept an empty array.